src/com/xilinx/rapidwright/RunCode.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2023, Advanced Micro Devices, Inc.
+ * All rights reserved.
+ *
+ * Author: Chris Lavin, AMD Research and Advanced Development.
+ *
+ * This file is part of RapidWright.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.xilinx.rapidwright;
+
+import java.io.IOException;
+
+import org.python.google.common.reflect.ClassPath;
+import org.python.google.common.reflect.ClassPath.ClassInfo;
+import org.python.util.jython;
+
+/**
+ * Convenience class to run arbitrary Java methods in RapidWright. Depends on
+ * the built-in Jython interpreter to run Jython/Java code.
+ *
+ */
+public class RunCode {
```

Comment:
Is `RunCode` the most precise? I would prefer `RunJythonCode`, `RunWithJython` instead, and consider eliminating all mention of Java entirely (e.g. `Java methods` or `Jython/Java code`)-- as I understand, what you're running is RapidWright through Python code; the underlying Python-Java translation is an implementation detail.

Reply:
Refactored to ``RunJython``.

Reply:
Is it worth merging the `RunJython` class into `Jython` (the latter was only added in #681) to reduce redundancy?

Reply:
I've tried and I couldn't seem to find a way to make it work.

Reply:
Sorry, I'm still a little lost on the difference between the two. Does `RunJython` now expected to do everything that `Jython` does?

Reply:
`Jython` is a wrapper for the Jython interpreter in that it will load a set of preselected RapidWright classes and return a prompt to the user.  If a user wants to run a single command, they would use the `-c` argument which shortcuts this pre-loading task.  

`RunJython` is mean to wrap the specific `jython -c` functionality.  It will not return a prompt to the user.  It is only designed to run a single code snippet from the command line.

Reply:
Ah, that makes sense. In that case, I would say that `Jython` with no arguments should be as it is today, but `Jython` with arguments (assumedly meaning that the user wants to run a single code snippet) would call what we now have as `RunJython`? Having two independent entrypoints called `Jython` and `RunJython` is potentially confusing.

Reply:
In the past, I had some complications with trying to modify args to the Jython interpreter.  However, I took another look and it seems like my changes are working now.